### PR TITLE
Add sqlite3 module

### PR DESCRIPTION
**Pull request by MichaelKim20**
Add sqlite3 module and test code
I ~will~ apply `asset function` to the test code.

### CODE REVIEW

File: tests/database.test.ts [hunk 1]
```diff
@@ -1,26 +1,51 @@
+/*******************************************************************************
+
+    Test for sqlite3 module.
+    The following functions are tested.
+    Create database, Add Records, Read All Records, Read Some Records, Close
+
+    Copyright:
+        Copyright (c) 2019-2020 BOS Platform Foundation Korea
```
Comment (TrustHenry) [inline tests/database.test.ts]:
```suggestion
        Copyright (c) 2020 BOS Platform Foundation Korea
```

**Review by TrustHenry — APPROVED** [review]
I think we can squash `tests/database.test.ts` into one commit.

File: package-lock.json [hunk 2]
```diff
@@ -1349,7 +1379,6 @@
       "version": "1.1.3",
       "resolved": "https://registry.npmjs.org/define-properties/-/define-properties-1.1.3.tgz",
       "integrity": "sha512-3MqfYKj2lLzdMSf8ZIZE/V+Zuy+BgD6f164e8K2w7dgnpKArBDerGYpM46IYYcjnkdPNMjPk9A6VFB8+3SKlXQ==",
-      "dev": true,
```
Comment (Geod24) [inline package-lock.json:1352]: What are all those `dev` changes ? They seem unrelated.

Comment (MichaelKim20) [reply package-lock.json:1352]: This file is automatically created when a dependency is installed.
There are two types of dependencies:
This is for development and product use.
Developmental use is an ancillary commercial module for development.
Product use is a required module when the product is running.
As sqlite3 was installed, the development was changed for the product.

File: tests/database.test.ts (hunk 1 above)
Comment (MichaelKim20) [reply tests/database.test.ts]: Thanks

**Comment by MichaelKim20** [comment]
Ready for review

File: tests/database.test.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    db.close();
+}
+
+function runChainExample ()
+{
+    createDb();
+}
+
+describe('Database', () => {
+    it('nothing', () => {
```
Comment (Geod24) [inline tests/database.test.ts]:
```suggestion
    it('Ensures that basic functionalities work', () => {
```

**Review by Geod24 — COMMENTED** [review]
Commits need to be squashed

File: tests/database.test.ts (hunk 3 above)
Comment (MichaelKim20) [reply tests/database.test.ts]: I changed it
